Misc/NEWS.d/next/Library/2020-05-02-01-01-30.bpo-29753.n2M-AF.rst

```diff
@@ -0,0 +1,3 @@
+Fixes packed bitfields not being calculated properly in some cases (see
+bpo-29753).
+Fixes packed bitfileds not being packed when created.
```

Comment:
Please fix the spelling in this sentence.

In my opinion, the news entry is rather vague currently and could use more detail.

Reply:
Yeah, I agree, this could use more detail. Keep in mind that this will be displayed with effectively no context other than the bpo issue, and bpo issues are often hard to read because there's a lot of back-and-forth trying to figure out the cause of the problem, what the solution is, etc. It's best to give a 1-2 sentence summary of what the problem is and what you changed to fix it.